mne/io/base.py

```diff
@@ -1850,9 +1850,10 @@ def plot_psd_topo(self, tmin=0., tmax=None, fmin=0, fmax=100, proj=False,
                                  show=show, block=block, n_jobs=n_jobs,
                                  axes=axes, verbose=verbose)
 
-    @fill_doc
+    @verbose
     def estimate_rank(self, tstart=0.0, tstop=30.0, tol=1e-4,
-                      return_singular=False, picks=None, scalings='norm'):
+                      return_singular=False, picks=None, scalings='norm',
+                      verbose=None):
```

Comment:
I would consider deprecating this method to avoid having 2 (inconsistant) ways of computing the rank from raw data.

Reply:
what do you think about this @larsoner ?

Reply:
Working on it now, then will rebase